scripts/tuple_tree_generator/tuple_tree_generator/templates/tuple_tree_gen.ts.tpl

```diff
@@ -0,0 +1,203 @@
+/**
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import * as yaml from "yaml"
+
+type TupleTreeRoot = {{ metadata.root }}
+
+{{ load_file('tuple_tree_gen.prelude.ts') }}
```

Comment:
I'm not really a fan of this, isn't there a cleaner way to do this?

Reply:
Split out the prelude, now it's a standalone `tuple_tree.ts`